analytics/src/test/java/com/segment/analytics/StatsTest.kt

```diff
@@ -0,0 +1,114 @@
+package com.segment.analytics
+
+import android.util.Pair
+import org.assertj.core.api.Assertions
+import org.assertj.core.data.MapEntry
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import java.io.IOException
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE)
+class StatsTest {
+    private var stats: Stats? = null
```

Comment:
same comment here. Is there a way to use `Stats` instead of `Stats?`. In these test scenarios we can confirm that this object is never null so this verbosity is kinda off-putting